<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BDTranslator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="numberLabel">
      <property name="text">
       <string>Number in the decimal system</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="numberSpinBox"/>
    </item>
    <item>
     <widget class="QLabel" name="precisionLabel">
      <property name="text">
       <string>Precision in the binary system</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="precisionSpinBox"/>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
